src/jit-diff/diff.cs

```diff
@@ -27,6 +20,58 @@ public enum DasmWorkKind
         Diff
     }
 
+    public class ProgressBar
```

Comment:
Fancy :-)

IMO, this might be "too much" progress output. seems like maybe once per second would be fine.

Reply:
I tried it at 5s and 1s initially and liked 0.1s better... Especially early, on when it seems not much is happening as we do some of the larger assemblies first.

Reply:
Maybe it should be progressive, 0.1 for the first 2 seconds, then 1s seconds after that?

Reply:
Give this a try, and if it's annoying, we can work on improving it.